packages/main/src/plugin/recommendations/recommendations-settings.ts

```diff
@@ -19,4 +19,5 @@
 export enum RecommendationsSettings {
   SectionName = 'extensions',
   IgnoreRecommendations = 'ignoreRecommendations',
+  IgnoreBannerRecommendations = 'ignoreBannerRecommendations',
```

Comment:
sidenote, not related to this PR, but we should try avoid repetition in enum, the enum is named `RecommendationsSettings` we can probably simplify `IgnoreRecommendations` to `IgnoreAll`, and `IgnoreBannerRecommendations` to `IgnoreBanner`.

(Let's keep this for now, as we don't want to change the `IgnoreRecommendations` in this PR)

Reply:
as long as the value is not changed yes we can make variable shorter 👍 (and of course in other PRs)